src/disco/css/Addon.scss

```diff
@@ -8,23 +8,33 @@ $secondary-text-color: #6a6a6a;
   overflow: hidden;
 
   .theme-image {
+    display: block;
     width: 100%;
   }
 
+  .install-button {
+    margin: 0 20px 0 30px;
```

Comment:
This will need an RTL override as well but I can't think of a way of doing this in flex. I'm not crazy about the padding on this block, nothing is consistent.

```
+-------------------------------------------------------+
|                       | 30px  |                       |
| 15px | <img /> | 35px | <p /> | 50px | <btn /> | 20px |
|                       | 30px  |                       |
+-------------------------------------------------------+